doc/adr/0007-deprecate-servicesync.md

```diff
@@ -0,0 +1,48 @@
+# 7. Deprecate ServiceSync
+
+Date: 2024-01-16
+
+## Status
+
+Proposed
+
+Enabled By [6. Explicit resources for declaring service bindings](0006-explicit-resources-for-declaring-service-bindings.md)
+
+## Context
+
+ServiceSync is a mechanism whereby service ingress bindings are
+automatically created at each site in the network for any service with
+egress bindings.
+
+Though this feature seemed nice in early demos, in practice it is
+generally unnecessary. In some cases it is explicitly not desirable as
+the ingress bindings should only be created in particular sites.
+
+Though the feature can at present be turned off, the option to do so
+is not obvious. Further it is not well tested.
+
+If the decision to have explicit resources for ingress bindings is
+adopted, it becomes even simpler to explicitly add the required
+ingresses to any site that requires them. Indeed as that proposal
+allows more flexibility with regard to the ingress bindings
+ServiceSync would not be adequate as it is.
+
+## Decision
+
+ServiceSync will not be supported for automatic propoagation of
+ingress bindings between sites that have migrated to use explicit
+resources for each ingress binding.
+
+## Consequences
```

Comment:
To me, the biggest consequence is the change to the mental model that we currently tell users about in docs.
However, this ADR only addresses the removal of the current model, and ADR 8 hints at the new model without defining it, but how about adding the following:

This decision removes the idea that users can expose a service on one site and then automatically consume that service on all other sites.

Reply:
This ADR only proposes the removal of a current feature. It doesn't in my view propose a different model per se. I agree that this is closely related to the decision proposed in ADR 6, and indeed they are linked together using the adr toolings linking support.